File: exercism/csharp/resistor-color/ResistorColor.cs
using System;
using System.Collections.Generic;

public static class ResistorColor
{
    private static List<string> ResistorsColors = new List<string>
    {
        "black",
        "brown",
        "red",
        "orange",
        "yellow",
        "green",
        "blue",
        "violet",
        "grey",
        "white"
    };

    public static int ColorCode(string color) => ResistorsColors.IndexOf(color);

    public static string[] Colors() => ResistorsColors.ToArray();
}

File: exercism/javascript/resistor-color.js
export const colorCode = (color) => {
  for(var i = 0; i < COLORS.length; i++)
  {
    if (COLORS[i] === color) return i;
  }
};

export const COLORS = ["black","brown","red","orange","yellow","green","blue","violet","grey","white"];

